packages/manager/apps/web/client/app/hosting/database/order/public/components/steps/db-categories-offers/component.js

```diff
@@ -8,6 +8,7 @@ const component = {
     preselectDbCategory: '<',
     onDbCategoryClick: '&',
     onDbCategoryEngineClick: '&',
+    model: '=',
```

Comment:
```suggestion
    model: '<',
```
IMO, it's better to use one way binding for input and output as a two way data binding for performance. In this case, the data is changed twice :
- 1 with the 2 way data binding
- 2 with the onDbCategoryClick event